lib/mongoid/changeable.rb

```diff
@@ -64,7 +64,7 @@ def changed_attributes
     #
     # @since 2.4.0
     def changes
-      _changes = {}
+      _changes = ActiveSupport::HashWithIndifferentAccess.new
```

Comment:
why don't we instead do the following on line 72:

```ruby
_changes.with_indifferent_access
```
to be consistent with the rest of the codebase?